### PR TITLE
Add 10 second timeout to all HTTP clients

### PR DESCRIPTION
**Pull request by julienduchesne**
Otherwise, calls can hang forever

### CODE REVIEW

File: pkg/grafana/httpclient.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package grafana
+
+import (
+	"net/http"
+	"time"
+)
+
+func NewHttpClient() *http.Client {
+	return &http.Client{Timeout: 10 * time.Second}
```
Comment (dannykopping) [inline pkg/grafana/httpclient.go:9]: Would be even nicer if this were configurable

Comment (sh0rez) [reply pkg/grafana/httpclient.go:9]: I second this, might render grizzly unusable in bad network conditions otherwise

Comment (malcolmholmes) [reply pkg/grafana/httpclient.go:9]: `GRIZZLY_HTTP_TIMEOUT` with a default of 10s?

Comment (dannykopping) [reply pkg/grafana/httpclient.go:9]: Seems reasonable to me

Comment (julienduchesne) [reply pkg/grafana/httpclient.go]: Done

**Review by dannykopping — APPROVED** [review]
Nice one, thanks @julienduchesne :+1:

**Review by malcolmholmes — APPROVED** [review]

**Pull request merged**